#request
bool isEndEffectorEnabled
bool endEffectorState
---
#response
bool success